src/app/shared/dialogs/dialogs-list.service.ts

```diff
@@ -51,7 +51,12 @@ export class DialogsListService {
   getListAndColumns(db: string, selector?: any, planetField: string = 'local') {
     selector = selector || this.defaultSelectorFunctions()[db];
     if (db === '_users') {
-      return this.couchService.findAll('_users', findDocuments({}, this.userService.userProperties))
+      return this.couchService.findAll('_users', findDocuments({
```

Comment:
@singharpita This users for communities are found in the `child_users` database on the nation.  This change will not affect the users from communities which have not been accepted.

We need to check the `communityregistrationrequests` database and filter if the community matching the `planetCode` has not been accepted when running `getListAndColumns` for `child_users`

Reply:
@paulbert child_users will not be synced until we approve community right. But we do send users to users database. And list is checking planetcode from both _users and child_users to show list.